docs/contributor/benchmarks/README.md

```diff
@@ -118,9 +118,11 @@ A typical test result output looks like the following example:
 |            0.107.0 |            19876            |            19719            |          0          |       184, 96        |     1, 1      |            11993             |            35504            |          0          |       133, 138       |   1.4, 1.4    |                   10081                    |             360             |         512         |      1734, 1722      |   0.5, 0.5    |                   18184                    |             922             |         510         |      1754, 1698      |    0.9, 1     |
 |            0.108.1 |            20175            |            20014            |          0          |       100, 93        |     1, 1      |            12365             |            36603            |          0          |       135, 148       |   1.4, 1.4    |                   10758                    |             332             |         512         |      1722, 1681      |   0.5, 0.5    |                   19163                    |             891             |         510         |      1800, 1757      |     1, 1      |
 
-
 </div>
 
+> **NOTE:** The expected throughput for the TracePipeline receiver is ~20K spans/sec for single pipeline scenario in other hand for multi pipeline scenario the expected throughput is ~13K spans/sec on receiver side and exported data should be 3 time of received spans. The backpressure scenarios should return the exporter queue size as ~512 for both single and multi pipeline scenarios and the TracePipeline instances should not crash.
+> Typical memory usage for TracePipeline instances is in average for single pod ~150MB and CPU usage is ~1.0 for non-backpressure scenarios, with backpressure scenarios the expected memory usage is significantly higher than non-backpressure scenarios on CPU side it should be less than non-backpressure scenarios since fewer data processed and exported compare to non-backpressure scenario. 
```

Comment:
```suggestion
> On average, memory usage for TracePipeline instances is ~150MB for a single Pod, and CPU usage is ~1.0 for non-backpressure scenarios.
> In backpressure scenarios, the behaviour is different: The expected CPU usage should be less than in non-backpressure scenarios, because less data is processed and exported compared to non-backpressure scenarios. The expected memory usage is significantly higher than in non-backpressure scenarios. 
```